openwisp_utils/staticfiles.py

```diff
@@ -12,7 +12,7 @@ class DependencyFinder(FileSystemFinder):
     A static files finder that finds static files of
     django-apps listed in dependencies
     """
-    dependencies = EXTENDED_APPS
+    dependencies = EXTENDED_APPS + ['openwisp_utils']
```

Comment:
It seems, that it's an easy fix to "putting `openwisp_utils` in `INSTALLED_APPS` problem"